Source/PersonalityInsightsV3/Models/ConsumptionPreferencesNode.swift

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import RestKit
+
+/** Provides every individual preference for a category. */
+public struct ConsumptionPreferencesNode: JSONDecodable {
+    
+    /// The globally unique id of the consumption preference.
+    public let consumptionPreferenceId: String
+    
+    /// The user-displayable name of the consumption preference.
+    public let name: String
+    
+    /// The score indicating the author's likelihood of preferring
+    /// the item. For example, if the score is 0.0, preference is
+    /// unlikely; if the score is 0.5, preference is neutral; if the score
+    /// is 1.0, the author would likely prefer the item.
+    public let score: Double
```

Comment:
The release notes specify that "the scores for some preferences are binary and do not allow a neutral value." I'm a little confused by that statement, because I'm not sure if it means the `score` will sometimes represented as a boolean (`true` or `false`) instead of a number (`1.0` or `0.0`).

Do you know if this value is always a number? I would assume that it is always a number, but it might be helpful to verify.

Since this value isn't optional, an error will be thrown if it is a boolean value instead of a number. If an error is thrown, then the optional might propagate up the model, making the `consumptionPreferences` property of the `Profile` model to be `nil`. It doesn't look like there is a test that verifies whether the `consumptionPreferences` is ever not `nil`, so that might be useful to add.